password-update-service-samples/README.md

```diff
@@ -0,0 +1,21 @@
+# Password Update Service Samples
+
+This repository contains sample implementations of password update services for [Asgardeo](https://console.asgardeo.io/) and [WSO2 Identity Server](https://wso2.com/identity-server/). These samples demonstrate how to create and configure a password update mechanism as an external service extension to the product.
+
+## Table of Contents
+- [Overview](#overview)
+- [Samples](#samples)
+- [Getting Started](#getting-started)
+
+## Overview
+The password update service samples in this repository are designed to help developers understand how to extend the password validation capabilities of Asgardeo and WSO2 Identity Server. Each sample provides an integration with [HaveIBeenPwned (HIBP)](https://haveibeenpwned.com/) and can be used as a reference for building your own custom password update services. The password update service can be used to enhance security by preventing users from using compromised passwords.
+
+## Samples
+### Express-based HIBP Service
```

Comment:
Shall we say HIBP Password Validator Service with Express ?

Reply:
Fixed with 0a6405027d81ee3b63b9bc16b80ededc1973d230